runhouse/resources/hardware/cluster.py

```diff
@@ -577,10 +577,14 @@ def check_server(self, restart_server=True):
         return
 
     def status(self):
-        self.check_server()
-        if self.on_this_cluster():
-            return obj_store.status()
-        return self.client.status()
+        try:
+            self.check_server()
+            if self.on_this_cluster():
+                return obj_store.status()
+            return self.client.status()
+        except ValueError as e:
+            print(str(e))
```

Comment:
raise the error instead of printing?
```suggestion
            raise e 
```